discovery_engine_api_tests/tests/test_back_office.py

```diff
@@ -0,0 +1,38 @@
+
+import allure
+import pytest
+from handlers.api_handler import ApiHandler
+from model import documents
+from utils import test_utils as su
+from utils import assert_utils as au
+
+
+@allure.suite("Back Office Test Suite")
+class TestBackOfficeEndpoint:
+    api_handler = ApiHandler()
+
+    @pytest.fixture
+    def generate_post_docs(self):
```

Comment:
:art: It unclear here if its "generate then post docs" or "generate [post docs]", but more important it's better to name it after what it does. E.g. `ingest_generated_documents`.